src/test/scala/sbtghactions/GenerativePluginSpec.scala

```diff
@@ -43,13 +43,14 @@ class GenerativePluginSpec extends Specification {
         |    branches: [main]
         |
         |jobs:
-        |  """.stripMargin
+        |${" " * 2}
```

Comment:
not proud of this, but given _some people_'s editors trim trailing whitespace automatically this is one _relatively_ reasonable way to include the whitespace. Alternatively we could exclude it from these tests... or revert this part and add `+ "\n"` at the end.

Reply:
I think this is a halfway decent hack actually